index.js

```diff
@@ -228,16 +228,15 @@ async function decorateRequestAndFastifyInstance(fastify, { asyncInitFunction })
       reply.send(new Error('Something went wrong'))
       return
     }
-    if (error.validation && error.validation.length > 0) {
-      const [{ instancePath, message }] = error.validation
-      const objectPath = `body${instancePath.replace(/\//g, '.')}`
-      const customErr = new Error(`${objectPath} ${message}`)
-      customErr.statusCode = 400
-      reply.send(customErr)
-      return
-    }
     reply.send(error)
   })
+  fastify.setSchemaErrorFormatter((errors, dataVar) => {
```

Comment:
what is this?

Reply:
A fastify function https://www.fastify.io/docs/latest/Server/#setschemaerrorformatter

Reply:
I mean, I knew that, the actual question was more like: what is this applied to? only validation errors for both request and response?